test/system/src/main/java/io/pravega/test/system/framework/services/kubernetes/AbstractService.java

```diff
@@ -77,8 +81,7 @@
     static final String ZOOKEEPER_OPERATOR_IMAGE = System.getProperty("zookeeperOperatorImage", "pravega/zookeeper-operator:latest");
     static final String IMAGE_PULL_POLICY = System.getProperty("imagePullPolicy", "Always");
     private static final String DOCKER_REGISTRY =  System.getProperty("dockerRegistryUrl", "");
-    private static final String PRAVEGA_VERSION = System.getProperty("imageVersion", "latest");
```

Comment:
Is the Pravega Operator going to work with the tags used in our snapshots? That is, using a tag like `0.6.0-2291.dd02097` is valid with the current upgrade logic?

Reply:
Yes @RaulGracia  those tags will work with the updated logic